readthedocs/core/utils/__init__.py

```diff
@@ -111,38 +111,9 @@ def trigger_build(project, version=None, record=True, force=False, basic=False):
 
 def send_email(recipient, subject, template, template_html, context=None,
                request=None):
-    '''
-    Send multipart email
-
-    recipient
-        Email recipient address
-
-    subject
-        Email subject header
-
-    template
-        Plain text template to send
-
-    template_html
-        HTML template to send as new message part
-
-    context
-        A dictionary to pass into the template calls
-
-    request
-        Request object for determining absolute URL
-    '''
```

Comment:
Also, generally I think it's a better pattern to have a util function that does the thing, and wrap it with the task. If we want to import a "send email" thing, we should import the util, not the task, IMO.